Formula/dumb.rb

```diff
@@ -0,0 +1,17 @@
+class Dumb < Formula
+  desc "IT, XM, S3M and MOD player library"
+  homepage "http://dumb.sourceforge.net/index.php?page=about"
+  url "https://downloads.sourceforge.net/project/dumb/dumb/0.9.3/dumb-0.9.3.tar.gz"
+  sha256 "8d44fbc9e57f3bac9f761c3b12ce102d47d717f0dd846657fb988e0bb5d1ea33"
+
+  def install
+    (buildpath/"make/config.txt").write <<-EOS.undent
+include make/unix.inc
+ALL_TARGETS := core core-examples core-headers
+PREFIX := #{prefix}
+    EOS
+    system "make"
+    mkdir_p [bin, include, lib]
```

Comment:
I know it doesn't necessary make a lot of sense in terms of compactness but we tend to use:
```ruby
bin.mkpath
include.mkpath
lib.mkpath
```